src/pages/studyView/table/CNAGenesTable.tsx

```diff
@@ -222,16 +222,30 @@ export class CNAGenesTable extends React.Component<ICNAGenesTablePros, {}> {
             headerRender: () => {
                 return <div style={{marginLeft: this.cellMargin[ColumnKey.FREQ]}}>Freq</div>
             },
-            render: (data: CopyNumberCountByGene) => <span
-                style={{
-                    flexDirection: 'row-reverse',
-                    display: 'flex',
-                    marginRight: this.cellMargin[ColumnKey.FREQ]
-                }}>{getFrequencyStr(data.frequency)}</span>,
-            sortBy: (data: CopyNumberCountByGene) => data.frequency,
+            render: (data: CopyNumberCountByGene) => {
+                const addTotalProfiledOverlay = () =>
+                    <span>{`# of samples profiled for CNA in this gene: ${data.numberOfSamplesProfiled}`}</span>;
+               return(
+                <div className={styles.displayFlex}>
+                    <DefaultTooltip
+                        placement="right"
+                        overlay={addTotalProfiledOverlay}
+                        destroyTooltipOnHide={true}
+                    >
+                        <span
+                            style={{
+                            flexDirection: 'row-reverse',
```

Comment:
i also wonder about this application of flex.  i can see it's old code, but there are a lot of cases of unnecessary application of flexbox in our app so i'm trying to get people to clean them up when it turns out they aren't necessary.

Reply:
I was more or less copy-pasting. I can take it out and see what happens.

Reply:
@alisman It seems to work fine without the flex box, so I took it out.

Reply:
@lemccarthy did you take out? i still see it here